Workbooks/Azure Monitor - Getting Started/Getting Started Resources/Helpful Links.workbook

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Getting started resources\r\n\r\nHere are some helpful links to find out more about certain features of Workbooks!\r\n- [Parameters](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Parameters/Parameters.md)\r\n- [Data Sources](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/DataSources.md)\r\n- [Resource Centric Logs](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/ResourceCentricLogs.md)\r\n- [Visualizations](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Visualizations/Visualizations.md)\r\n- [JSONPath](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Transformations/JSONPath.md)\r\n- [Links](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Links/Links.md)\r\n- [Interactive Workbooks](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Interactivity.md)\r\n- [Programatically manage Workbooks](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Programmatically.md)\r\n\r\n## Samples\r\nBrowse some of our samples [here](https://github.com/microsoft/Application-Insights-Workbooks/tree/master/Documentation/Samples)!\r\n\r\n## Are you coming from View Designer?\r\nHere's how you can make the shift to Workbooks!\r\n- [View Designer to Workbooks Transition Guide](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/ViewDesignerOverview.md)\r\n- [Accessing Workbooks & Viewing Permissions](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/AccessPermissions.md)\r\n- [Common Steps](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/CommonSteps.md)\r\n- [Conversion Options](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/ConversionOptions.md)\r\n- [Tile Conversions](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/ViewDesigner/TileConversions.md)\r\n\r\n---\r\n\r\nFind out more about [contributing](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Contributing.md) to Workbooks!\r\n"

```

Comment:
my only nit here would be to use the docs on azure docs if they exist first, falling back to our github ones?  the azure docs are supposed to be our "official" ones, and the github ones kindof the "developer" ones?  this is a good start though!